src/Formatter/CurlCommandFormatter.php

```diff
@@ -36,7 +36,10 @@ public function formatRequest(RequestInterface $request)
 
         $body = $request->getBody();
         if ($body->getSize() > 0) {
-            if ($body->isSeekable()) {
+            // escapeshellarg argument max length on Windows, but impractical to use either way
```

Comment:
maybe say "... max length on Windows, but longer body in curl command would be impractical anyways."

Reply:
Updated :)